examples/webdriver.java

```diff
@@ -27,8 +29,10 @@ public static WebDriver setupDriver() {
     options.addArguments("--disable-dev-shm-usage");
     options.addArguments("--disable-browser-side-navigation"); 
     options.addArguments("--disable-gpu");
-    options.setPageLoadStrategy(PageLoadStrategy.NONE); 
-    return new ChromeDriver(options); 
+    options.setPageLoadStrategy(PageLoadStrategy.NONE);
+    WebDriver driver = new ChromeDriver(options);
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

Comment:
at first I thought this was adding a delay but actually it doesn't - it makes it wait the just right amount and times out if nothing after 10 seconds. Good fix. I pushed it!